<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayWidget</class>
 <widget class="QWidget" name="DisplayWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>498</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lblStatusDW">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Status:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblRecordingDW">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>400</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Not Recording</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblFileNameDW">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>400</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Filename</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editFileNameDW">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>400</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lblFPSDW">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>400</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>FPS</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="sbDW">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>400</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbRecordDW">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>450</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Record</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbPreserveDW">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>450</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Preserve Aspect Ratio for Display</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDisplay">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>441</width>
     <height>321</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
